src/backend/joanie/core/enums.py

```diff
@@ -77,6 +77,7 @@
     ORDER_STATE_PENDING,
     ORDER_STATE_VALIDATED,
 )
+MIN_ORDER_TOTAL_DEFAULT = 0.0
```

Comment:
```suggestion
MIN_ORDER_TOTAL_AMOUNT
```

A default means it can be modified, an other value can be provided using a settings for example. It is not the case here.